packages/ods/src/components/quantity/src/components/ods-quantity/ods-quantity.tsx

```diff
@@ -0,0 +1,146 @@
+import { type OdsQuantityEventValueChangeDetail } from '../../interfaces/events';
+
+@Component({
+  formAssociated: true,
+  shadow: true,
+  styleUrl: 'ods-quantity.scss',
+  tag: 'ods-quantity',
+})
+export class OdsQuantity {
+  private odsInput?: OdsInput;
+
+  @AttachInternals() internals!: ElementInternals;
+
+  @Prop({ reflect: true }) public ariaLabel: HTMLElement['ariaLabel'] = null;
+  @Prop({ reflect: true }) public ariaLabelledby?: string;
+  @Prop({ reflect: true }) public defaultValue?: number;
+  @Prop({ reflect: true }) public hasError: boolean = false;
+  @Prop({ reflect: true }) public isDisabled: boolean = false;
+  @Prop({ reflect: true }) public isReadonly: boolean = false;
+  @Prop({ reflect: true }) public isRequired: boolean = false;
+  @Prop({ reflect: true }) public max?: number;
+  @Prop({ reflect: true }) public min?: number;
+  @Prop({ reflect: true }) public name!: string;
+  @Prop({ reflect: true }) public placeholder?: string;
+  @Prop({ reflect: true }) public step?: number;
+  @Prop({ mutable: true, reflect: true }) public value: number | null = null;
+
+  @Event() odsBlur!: EventEmitter<void>;
+  @Event() odsChange!: EventEmitter<OdsQuantityEventValueChangeDetail>;
+  @Event() odsClear!: EventEmitter<void>;
+  @Event() odsFocus!: EventEmitter<void>;
+  @Event() odsReset!: EventEmitter<void>;
+
+  @Method()
+  async clear(): Promise<void> {
+    return this.odsInput?.clear();
+  }
+
+  @Method()
+  async getValidity(): Promise<ValidityState | undefined> {
+    return this.odsInput?.getValidity();
+  }
+
+  @Method()
+  async reset(): Promise<void> {
+    return this.odsInput?.reset();
+  }
+
+  componentWillLoad(): void {
+    if (!this.value && this.value !== 0) {
+      this.value = this.defaultValue ?? null;
+    }
+    setFormValue(this.internals, this.value);
+  }
+
+  async formResetCallback(): Promise<void> {
+    await this.reset();
+  }
+
+  private decrement(): void {
+    if (this.isDisabled || this.isReadonly) {
+      return;
+    }
+    const step = this.step || 1;
+    this.value = this.value !== null ? Number(this.value) - step : 0;
+  }
+
+  private increment(): void {
+    if (this.isDisabled || this.isReadonly) {
+      return;
+    }
+    const step = this.step || 1;
+    this.value = this.value !== null ? Number(this.value) + step : 0;
+  }
+
+  private onOdsChange(event: OdsInputValueChangeEvent): void {
+    if (event.detail.value === null) {
+      this.value = null;
+      return;
+    }
+    this.value = Number(event.detail.value) ?? null;
+    setFormValue(this.internals, this.value);
+  }
+
+  render(): FunctionalComponent {
+    return (
+      <Host class="ods-quantity">
+        <ods-button
+          class={{
+            'ods-quantity__button': true,
+            'ods-quantity__button--readonly': this.isReadonly,
+          }}
+          color={ this.hasError ? ODS_BUTTON_COLOR.critical : ODS_BUTTON_COLOR.primary }
+          exportparts="button:button_minus"
```

Comment:
`button-minus` (classic case for css)

Reply:
In the mozilla doc the name is like this https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/exportparts#exposing_mapped_parts

Reply:
it's a double `_` on the doc, probably from a BEM exemple, part are used on css side, so generic case would be with a `-`